### PR TITLE
Dev task/convert controllers to es6

### PR DESCRIPTION
**Pull request by bendiggle**

### CODE REVIEW

File: apps/gro/controllers/post.js [hunk 1]
```diff
@@ -1,33 +1,25 @@
...
-/* eslint no-unused-vars: 0 */
-PostController.prototype.saveValues = function saveValues(req, res, callback) {
+module.exports = class PostController extends BaseController {
+  constructor(options) {
+    super(options);
+  }
 
-  var address = (function createAddressString(data) {
+  saveValues(req, res, callback) {
+    const address = ((data) => {
```
Comment (JoeChapman) [inline apps/gro/controllers/post.js]: You don't need the inner parenthesis around the function arguments.

File: apps/common/controllers/start.js [hunk 2]
```diff
@@ -1,18 +1,16 @@
...
+    super(options);
+  }
 
-StartController.prototype.getValues = function getValues(req) {
-  req.sessionModel.reset();
-  BaseController.prototype.successHandler.apply(this, arguments);
-};
+  getValues(req, res, callback) {
+    req.sessionModel.reset();
+    super.successHandler(req, res, callback);
```
Comment (JoeChapman) [inline apps/common/controllers/start.js]: It's purely personal preference, but I prefer `super.successHandler.apply(this, arguments);` I also think that there is less opportunity for any of the arguments to be accidentally omitted

File: apps/gro/controllers/post.js (hunk 1 above)
Comment (bendiggle) [inline apps/gro/controllers/post.js]: Wow! That's cool. Didn't know that!

File: apps/common/controllers/start.js (hunk 2 above)
Comment (bendiggle) [inline apps/common/controllers/start.js]: So I've done it the way you suggested but it meant having to put `/*eslint no-unused-vars: 0*/` before getValues so the lint doesn't get upset about the un-used variables

**Comment by bendiggle** [comment]
@JoeChapman This has been updated now

**Pull request merged**